x-pack/plugins/infra/server/lib/alerting/metric_threshold/lib/evaluate_alert.ts

```diff
@@ -206,21 +200,18 @@ const getMetric: (
         bucketSelector,
         afterKeyHandler
       )) as Array<Aggregation & { key: Record<string, string>; doc_count: number }>;
-      const groupedResults = compositeBuckets.reduce(
-        (result, bucket) => ({
-          ...result,
-          [Object.values(bucket.key)
-            .map((value) => value)
-            .join(', ')]: getValuesFromAggregations(
-            bucket,
-            aggType,
-            dropPartialBucketsOptions,
-            calculatedTimerange,
-            bucket.doc_count
-          ),
-        }),
-        {}
-      );
+      const groupedResults: Record<string, any> = {};
```

Comment:
Is it possible to use `Record<string, ReturnValue<typeof getValuesFromAggregations>>` here to help with that `any` typing...

Reply:
Fixed

Reply:
:chefs-kiss: thanks!